mage_ai/services/k8s/config.py

```diff
@@ -18,32 +22,78 @@ class K8sResourceConfig(BaseConfig):
 
 @dataclass
 class K8sExecutorConfig(BaseConfig):
-    container_config: Dict = None
```

Comment:
We shouldn't delete these fields. It'll break the current usage of the k8s executors.

Reply:
Done